app/views/content_items/statistical_data_set.html.erb

```diff
@@ -1,7 +1,41 @@
-<%= render 'shared/title_and_translations', content_item: @content_item %>
-<%= render 'components/notice', @content_item.withdrawal_notice_component  %>
-<%= render 'shared/metadata', content_item: @content_item %>
-<%= render 'shared/history_notice', content_item: @content_item %>
-<%= render 'govuk_component/lead_paragraph', text: @content_item.description %>
-<%= render 'shared/sidebar_contents_with_body', content_item: @content_item %>
-<%= render 'shared/footer', @content_item.document_footer %>
+<%= content_for :simple_header, @navigation.should_present_taxonomy_navigation? %>
+
+<div class="grid-row">
+    <div class="column-two-thirds">
+        <%= render 'govuk_component/title',
+            context: t("content_item.schema_name.#{@content_item.document_type}", count: 1),
```

Comment:
[If there are no different `document_type`s for statistical data sets](https://github.com/alphagov/govuk-content-schemas/blob/master/dist/formats/statistical_data_set/frontend/schema.json#L41) then we don't need to provide this `context` arg.

Reply:
@steventux Not sure I agree. Doesn't it need to still say "Statistical data set" above the title?

Reply:
ignore me, didn't understand the purpose of this correctly.